resources/assets/tconstruct/models/block/rack.json

```diff
@@ -0,0 +1,28 @@
+{
+    "__comment": "The model is in the center, top of the block. The model is translated instead of rotated when possible, to keep the top texture as a beam while lining up with planks",
+    "parent": "block/block",
+    "display": {
+        "gui": {
+            "rotation": [ 30, 225, 0 ],
+            "translation": [ 0, -3, 0 ],
+            "scale": [ 0.625, 0.625, 0.625 ]
+        }
+    },
+    "textures": {
+        "particle": "#bottom"
```

Comment:
Rename the #bottom texture to #leg, for better logical coupling with the table model. Also define the bottom/leg texture with a default here, or put a default texture into the blockstate, to remove the block models.